frontend/src/routes/(authenticated)/(dashboards)/admin/+page.svelte

```diff
@@ -102,24 +163,56 @@
                 <span class="i-mdi-sort-ascending text-xl align-[-5px] ml-2" />
               </th>
               <th>{$t('admin_dashboard.column_type')}</th>
+              <th />
             </tr>
           </thead>
           <tbody>
-            {#each projects as project}
+            {#each projects as project (project.id)}
               <tr>
                 <td>
-                  <a class="link" href={`/project/${project.code}`}>
-                    {project.name}
-                  </a>
+                  {#if project.deletedDate}
+                    <span class="flex gap-2 text-error">
+                      {project.name}
+                      <TrashIcon pale />
+                    </span>
+                  {:else}
+                    <a class="link" href={`/project/${project.code}`}>
+                      {project.name}
+                    </a>
+                  {/if}
                 </td>
                 <td>{project.code}</td>
                 <td>TODO</td>
                 <td>
-                  <FormatDate date={project.lastCommit} />
+                  {#if project.deletedDate}
+                    <span class="text-error">
+                      <FormatDate date={project.deletedDate} />
+                    </span>
+                  {:else}
+                    <FormatDate date={project.lastCommit} />
+                  {/if}
                 </td>
                 <td>
                   <FormatProjectType type={project.type} />
                 </td>
+                <td class="p-0">
+                  {#if !project.deletedDate}
+                    <div class="dropdown dropdown-end">
```

Comment:
should probably make a menu/dropdown component as we have one here and on the project page for editing members.